guides/tutorials/issue-a-credential.md

```diff
@@ -28,25 +28,28 @@ Depending on your target, [React
 Native](../getting-started/prerequisites/react-native) or
 [Node.js](../getting-started/prerequisites/react-native), it might vary.
 
-In this tutorial the _Holder_ will be in a [React Native
-environment](../getting-started/prerequisites/react-native) and the _Issuer_ in
-a [Node.js environment](../getting-started/prerequisites/nodejs).
+In this tutorial the _Holder_ will be in a [Node.js
+environment](../getting-started/prerequisites/nodejs) and the _Issuer_
+also in a [Node.js environment](../getting-started/prerequisites/nodejs).
 
 #### Holder
 
 For the _Holder_ we need to setup a basic agent with a wallet, mediator,
 outbound transport and a ledger.
 
-It is very important to note that mobile agents do not support HTTP by default.
-It is recommended to do everything over HTTPS, but for development HTTP can be
-enabled for
-[iOS](https://stackoverflow.com/questions/30731785/how-do-i-load-an-http-url-with-app-transport-security-enabled-in-ios-9)
-and
-[Android](https://stackoverflow.com/questions/51902629/how-to-allow-all-network-connection-types-http-and-https-in-android-9-pie).
+> If you want to follow this tutorial in a mobile environment:
+>
+> 1. Use the `agentDependencies` from `@aries-framework/react-native`
+> 1. It is very important to note that mobile agents do not support HTTP by default.
```

Comment:
Can we use [Admonitions](https://docusaurus.io/docs/markdown-features/admonitions) for this? Looks better IMO. Doesn't have to be this PR, maybe we can add a PR that changes this for all files at once

![image](https://user-images.githubusercontent.com/23165168/174590404-ab968cec-2aa6-48a1-981d-74264572409d.png)